recipe/meta.yaml

```diff
@@ -8,11 +8,11 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8b2579bf8568e32a4ad69be2810b300431d69f6691ed473f5aadb9471c7bc3bf
+  sha256: 9794a1560f2d3f22905853b5fd9db74018428bd47a366d5a894dee7362142cb0
 
 
 build:
-  number: 0
+  number: 1
```

Comment:
Increasing build number is only necessary when a package with the same version is already present in the channel. Since this PR add a new version, 0 is the correct value.